en/SUMMARY.md

```diff
@@ -6,7 +6,7 @@
   * [Toolchain Installation](setup/dev_env.md)
     * [Mac OS](setup/dev_env_mac.md)
     * [Linux](setup/dev_env_linux.md)
-      * [Advanced Linux](setup/dev_env_linux_boutique.md)
+      * [Arch/CentOS Linux](setup/dev_env_linux_boutique.md)
```

Comment:
FYI, renamed to make it clear what linux is covered. This is not "Advanced topics, it is just "using Arch/Cent OS.